easybuild/easyconfigs/p/phyx/phyx-1.01_link_scalapack.patch

```diff
@@ -0,0 +1,12 @@
+diff -Naur phyx-1.01.orig/src/Makefile.in phyx-1.01/src/Makefile.in
```

Comment:
@omula This patch should be removed, no longer used?

Reply:
@boegel you are right. done